code/feature_extraction/ImageHelper.py

```diff
@@ -3,26 +3,34 @@
 except ImportError:
     import Image
 import pytesseract
-import os
+from pathlib import Path
 
 # Make sure that Tesseract is defined even if it is not in $PATH variable
 pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract'

```

Comment:
I think we'd better check this in the notebook or somewhere else outside this file (to also not hardcode it)